README.md

```diff
@@ -1,3 +1,37 @@
 # Monitoring operator for storages
 
 This operator takes care of deployment of mixins of storage system in Openshift and Kubernetes. The mixins are the mechanism for adding alerting rules, grafana dashboards and recording rules for prometheus in Openshift and Kubernetes.
+
+# Deploy Operator with mixins
+To deploy the operator simply deploy to your cluster. The set of steps to be followed
+for deployment of monstorak operator and related artifacts, follow the below steps
```

Comment:
can we simplify the steps? Writing 5 different commands is a tedious task. If we can have single description and add these steps in format that a person can copy paste and done

Reply:
Ack